### PR TITLE
Changelog v1.13.0-RC1

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

**Comment by lunny** [comment]
rc1

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,321 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.13.0](https://github.com/go-gitea/gitea/releases/tag/v1.13.0) - 2020-10-14
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
## [1.13.0-RC1](https://github.com/go-gitea/gitea/releases/tag/v1.13.0-RC1) - 2020-10-14
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,321 @@ This changelog goes through all the changes that have been made in each release
...
+  * Remove page size limit comment from swagger (#11806)
+  * Narrow down Edge version in browser support docs (#11640)
+* MISC
+  * Use time/tzdata for timezone in docker  (#12922)
+  * Update gitea.com/macaron/i18n (#12806)
+  * Update JS dependencies (#12782)
+  * Upgrade xorm to v1.0.5 (#12765)
+  * Upgrade bleve to v1.0.10 (#12737)
+  * Update to go-org 1.3.2 (#12728)
+  * [Vendor] update go-swagger v0.21.0 -> v0.25.0 (#12670)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,320 @@ This changelog goes through all the changes that have been made in each release
...
+  * Return error when Mirrors are globaly disabled and like to create one (#11757)
+  * Provide diff and patch API endpoints (#11751)
+  * Allow to create closed milestones (#11745)
+  * Add language Statistics endpoint (#11737)
+  * Add Endpoint to get GetGenneralUI Settings (#11735) & (#11854)
+  * Issue/Pull expose IsLocked Property on API (#11708)
+  * Add endpoint for Branch Creation (#11607)
+  * Add pagination headers on endpoints that support total count from database (#11145)
+* BUGFIXES
+  * Update golangci-lint to version 1.31.0 (#13102)
```
Comment (6543) [inline CHANGELOG.md:75]:
```suggestion
  * Show original author's reviews on pull summary box (#13127)
  * Update golangci-lint to version 1.31.0 (#13102)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,320 @@ This changelog goes through all the changes that have been made in each release
...
+  * Narrow down Edge version in browser support docs (#11640)
+* MISC
+  * Use time/tzdata for timezone in docker  (#12922)
+  * Update gitea.com/macaron/i18n (#12806)
+  * Update JS dependencies (#12782)
+  * Upgrade xorm to v1.0.5 (#12765)
+  * Upgrade bleve to v1.0.10 (#12737)
+  * Update to go-org 1.3.2 (#12728)
+  * Vendor update: github.com/yuin/goldmark v1.2.1 (#12377)
+  * Upgrade xorm to v1.0.3 (#12210)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,320 @@ This changelog goes through all the changes that have been made in each release
...
+  * Update Goldmark v1.1.25 -> v1.1.32 (#12048)
+  * upgrade pq to v1.7.0 (#11905)
+  * upgrade to use testfixtures v3 (#11904)
+  * Upgrade xorm to v1.0.2 (#11900)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,320 @@ This changelog goes through all the changes that have been made in each release
...
+  * Alpine 3.12 (#11720)
+  * Enable stylelint's shorthand-property-no-redundant-values (#11436)
+* DOCS
+  * Change default log configuration (#13088)
+  * Add automatic JS license generation (#11810)
+  * Remove page size limit comment from swagger (#11806)
+  * Narrow down Edge version in browser support docs (#11640)
+* MISC
+  * Use time/tzdata for timezone in docker  (#12922)
+  * Update gitea.com/macaron/i18n (#12806)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Don't add same line code comment box twice (#11837)
+  * Fix visibility of forked public repos from private orgs (#11717)
+  * Fix chardet test and add ordering option (#11621)
+  * Fix number of files, total additions, and deletions on Diff pages (#11614)
+  * Properly handle and return empty string for dangling commits in GetBranchName (#11587)
+  * Include query in sign in redirect (#11579)
+  * Fix Enter not working in SimpleMDE (#11564)
+  * Fix bug about can't skip commits base on base branch (#11555)
+* ENHANCEMENTS
+  * Save TimeStamps for Star, Label, Follow, Watch and Collaboration to Database (#13124)
```
Comment (6543) [inline CHANGELOG.md:147]:
```suggestion
  * Support storing User-, Org- Repo-Avatars in minio (#12516)
  * Save TimeStamps for Star, Label, Follow, Watch and Collaboration to Database (#13124)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Git migration UX improvements (#12619)
+  * Add link to home page on swagger ui (#12601)
+  * hCaptcha Support (#12594)
+  * OpenGraph: use repo avatar if exist (#12586)
+  * Reaction picker display improvements (#12576)
+  * Fix emoji replacements, make emoji images consistent (#12567)
+  * Increase clickable area on files table links (#12553)
+  * Set z-index for sticky diff box lower (#12537)
+  * Report error if API merge is not allowed (#12528)
+  * LFS support to be stored on minio (#12518)
```
Comment (6543) [inline CHANGELOG.md:184]:
```suggestion
  * LFS support to be stored on minio (#12518)
  * Support storing User-, Org- Repo-Avatars in minio (#12516)
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Change PIDFile default from /var/run/gitea.pid to /run/gitea.pid (#12500)
+  * Add extension Support to Attachments (allow all types for releases) (#12465)
+  * Remove IE11 Support (#11470)
+* FEATURES
+  * Adopt repositories (#12920)
+  * Set setting.AppURL as GITEA_ROOT_URL environment variable during pushes (#12752)
+  * Only use async pre-empt hack if go < 1.15 (#12718)
+  * Check passwords against HaveIBeenPwned (#12716)
+  * Gitea 2 Gitea migration (#12657)
+  * Add postgres schema to the search_path on database connection (#12634)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Add postgres schema to the search_path on database connection (#12634)
  * Support storing Avatars in minio  (#12516)
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Set TLS minimum version to 1.2 (#12689)
+  * Add Primary Key to Topic and RepoTopic tables (#12639)
+  * Disable password complexity check default (#12557)
+  * Change PIDFile default from /var/run/gitea.pid to /run/gitea.pid (#12500)
+  * Add extension Support to Attachments (allow all types for releases) (#12465)
+  * Remove IE11 Support (#11470)
+* FEATURES
+  * Adopt repositories (#12920)
+  * Set setting.AppURL as GITEA_ROOT_URL environment variable during pushes (#12752)
+  * Only use async pre-empt hack if go < 1.15 (#12718)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Simplify CheckUnitUser logic (#12854)
+  * Fix background of signed-commits on arc-green of timeline commits (#12837)
+  * Move git update-server-info to hooks (#12826)
+  * Add ui style for "Open a blank issue" button (#12824)
+  * Use a simple format for the big number on ui (#12822)
+  * Make SVG size argument optional (#12814)
+  * Add placeholder text for bio profile text form (#12792)
+  * Set language via AJAX (#12785)
+  * Show git-pull-request icon for closed pull request (#12742)
+  * [Vendor] Switch go-version lib (#12719)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Migrate version parsing library to hashicorp/go-version (#12719)
  * Only use async pre-empt hack if go < 1.15 (#12718)
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * GetReleaseByID return 404 if not found (#12933)
+  * Get release by tags endpoint (#12932)
+  * Not using "ctx.ServerError" in api (#12907)
+  * NotificationSubject show Issue/Pull State (#12901)
+  * Refactor: move Commit To APIFormat Code & Lot of StopWatch related things (#12729)
+  * Expose its limitation settings (#12714)
+  * Add Created & Updated to Milestone (#12662)
+  * Milestone endpoints accept names too (#12649)
+  * Expose Attachment Settings in the API (#12514)
+  * additional data for type StopWatch in swagger api (#12458)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Add Issue and Repo info to StopWatch (#12458)
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+* API
+  * If User is Admin, show 500 error message on PROD mode too (#13115)
+  * Add Timestamp to Tag list API (#13026)
+  * Return sample message for login error in api context (#12994)
+  * Add IsTemplate option in create repo ui and api (#12942)
+  * GetReleaseByID return 404 if not found (#12933)
+  * Get release by tags endpoint (#12932)
+  * Not using "ctx.ServerError" in api (#12907)
+  * NotificationSubject show Issue/Pull State (#12901)
+  * Refactor: move Commit To APIFormat Code & Lot of StopWatch related things (#12729)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 14]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Support elastic search for code search (#10273)
+  * Kanban board (#8346)
+* API
+  * If User is Admin, show 500 error message on PROD mode too (#13115)
+  * Add Timestamp to Tag list API (#13026)
+  * Return sample message for login error in api context (#12994)
+  * Add IsTemplate option in create repo ui and api (#12942)
+  * GetReleaseByID return 404 if not found (#12933)
+  * Get release by tags endpoint (#12932)
+  * Not using "ctx.ServerError" in api (#12907)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 15]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Increase size of the language column in language_stat (#12396)
+  * Use transaction in V102 migration (#12395)
+  * Only use --exclude on name-rev with git >= 2.13 (#12347)
+  * Update chroma to v0.8.0 (#12337)
+  * Add action feed for new release (#12324)
+  * Set NoAutoTime when updating is_archived (#12266)
+  * Support Force-update in Mirror and improve Tracing in mirror (#12242)
+  * Avoid sending "0 new commits" webhooks (#12212)
+  * Fix U2F button icon (#12167)
+  * models: break out of loops (#12159)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * models/repo_sign.go: break out of loops (#12159)
```
@zeripath do we have a better name for this?

File: CHANGELOG.md [hunk 16]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Use label to describe repository type (#11891)
+  * Make repository size unclickable on repo summary bar (#11887)
+  * Rework blame template and styling (#11885)
+  * Fix icon alignment for show/hide outdated link on resolved conversation (#11881)
+  * Vertically align review icons on repository sidebar (#11880)
+  * Better align items using flex within review request box (#11879)
+  * Only write to global gitconfig if necessary (#11876)
+  * Disable all typographic replacements in markdown renderer (#11871)
+  * Improve label edit buttons labels (#11841)
+  * Refactor: Remove Dependencys from Migration v112-v119 (#11811)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 17]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Use crispEdges rendering for octicon-internal-repo (#11801)
+  * Work around fomantic 2.8.5 bugs (#11767)
+  * Show update branch item in merge box when it's necessary (#11761)
+  * Add compare link to releases (#11752)
+  * Allow site admin to disable mirrors (#11740)
+  * Export monaco editor on window.codeEditors (#11739)
+  * Update fomantic-ui to 2.8.5 (#11736)
+  * Refactor repo_stats to use os.pipe (#11726)
+  * bug: fix wrong poster message for code comment on Pull view (#11721)
```
Comment (6543) [inline CHANGELOG.md]: bugfix?

File: CHANGELOG.md [hunk 18]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add configurable Trust Models (#11712)
+  * Show full GPG commit status on PR commit history (#11702)
+  * Fix align issues and decrease avatar size on PR timeline (#11689)
+  * Replace jquery-datetimepicker with native date input (#11684)
+  * ui: change commentTag style (#11668)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Change Style of Tags on Comments (#11668)
```

File: CHANGELOG.md [hunk 19]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Refactor: Remove Dependencies from Migration v111 (#11646)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 20]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix missing styling for shabox on PR commit history (#11625)
+  * Apply padding to approval icons on PR list (#11622)
+  * Fix message wrapping on PR commit list (#11616)
+  * rework eslint config (#11615)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Rework eslint config (#11615)
```

File: CHANGELOG.md [hunk 21]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix styling and padding for commit list on PR view (#11588)
+  * Style code review comment for arc-green (#11572)
+  * Use default commit message for wiki edits (#11550)
+  * Add internal-repo octicon for public repos of private org (#11529)
+  * Fix dropzone color on arc-green (#11514)
+  * Insert ui divider directly in templates instead of from inside heatmap vue component (#11508)
+  * Move tributejs to npm/webpack (#11497)
+  * Fix text-transform on wiki revisions page (#11486)
+  * Do not show lock icon on repo list for public repos in private org (#11445)
+  * models: include LFS when calculating repo size (#11060)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Include LFS when calculating repo size (#11060)
```

File: CHANGELOG.md [hunk 22]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add check for LDAP group membership (#10869)
+  * When starting new stopwatch stop previous if it is still running (#10533)
+  * Add queue for code indexer (#10332)
+  * Move all push update operations to a queue (#10133)
+  * Cache last commit when pushing for big repository (#10109)
+  * Change/remove a branch of an open issue (#9080)
+  * [UI] Sortable Tables Header By Click (#7980)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Sortable Tables Header By Click (#7980)
```

File: CHANGELOG.md [hunk 23]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+* TESTING
+  * use community codecov drone plugin (#12468)
+  * Add more tests for diff highlighting (#12467)
+  * Don't put integration test data outside of test folder (#11746)
+  * Add debug option to hooks (#11624)
+  * log slow tests (#11487)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Log slow tests (#11487)
```

File: CHANGELOG.md [hunk 24]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Translate two small lables on commit statuse list (#12821)
+  * Make issues.force_push_codes message shorter (#11575)
+* BUILD
+  * Bump min required golang to 1.13 (#12717)
+  * Add 'make watch' (#12636)
+  * Extract Swagger CSS to its own file (#12616)
+  * Update eslint config (#12609)
+  * Avoid unnecessary system-ui expansion (#12522)
+  * Make the default PID file compile-time settable (#12485)
+  * go1.15 (#12475)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 25]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * Narrow down Edge version in browser support docs (#11640)
+* MISC
+  * Use time/tzdata for timezone in docker  (#12922)
+  * Update JS dependencies (#12782)
+  * Upgrade xorm to v1.0.5 (#12765)
+  * Upgrade bleve to v1.0.10 (#12737)
+  * Update to go-org 1.3.2 (#12728)
+  * Vendor update: github.com/yuin/goldmark v1.2.1 (#12377)
+  * Update Goldmark v1.1.25 -> v1.1.32 (#12048)
+  * upgrade pq to v1.7.0 (#11905)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Upgrade pq to v1.7.0 (#11905)
```

File: CHANGELOG.md [hunk 26]
```diff
@@ -4,6 +4,318 @@ This changelog goes through all the changes that have been made in each release
...
+  * upgrade to use testfixtures v3 (#11904)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Upgrade to use testfixtures v3 (#11904)
```

File: CHANGELOG.md [hunk 27]
```diff
@@ -4,6 +4,313 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add automatic JS license generation (#11810)
+  * Remove page size limit comment from swagger (#11806)
+  * Narrow down Edge version in browser support docs (#11640)
+* MISC
+  * Use time/tzdata for timezone in docker  (#12922)
+  * Update JS dependencies (#12782)
+  * Upgrade xorm to v1.0.5 (#12765)
+  * Upgrade bleve to v1.0.10 (#12737)
+  * Update to go-org 1.3.2 (#12728)
+  * Vendor update: github.com/yuin/goldmark v1.2.1 (#12377)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Update goldmark to v1.2.1 (#12377)
```

File: CHANGELOG.md [hunk 28]
```diff
@@ -4,6 +4,313 @@ This changelog goes through all the changes that have been made in each release
...
+  * Update Goldmark v1.1.25 -> v1.1.32 (#12048)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 29]
```diff
@@ -4,6 +4,313 @@ This changelog goes through all the changes that have been made in each release
...
+  * Move git update-server-info to hooks (#12826)
+  * Add ui style for "Open a blank issue" button (#12824)
+  * Use a simple format for the big number on ui (#12822)
+  * Make SVG size argument optional (#12814)
+  * Add placeholder text for bio profile text form (#12792)
+  * Set language via AJAX (#12785)
+  * Show git-pull-request icon for closed pull request (#12742)
+  * Migrate version parsing library to hashicorp/go-version (#12719)
+  * Only use async pre-empt hack if go < 1.15 (#12718)
+  * UI: Inform user what meaning hourglass on reviews has (#12713)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Inform user about meaning of an hourglass on reviews (#12713)
```

File: CHANGELOG.md [hunk 30]
```diff
@@ -4,6 +4,313 @@ This changelog goes through all the changes that have been made in each release
...
+  * Improve label edit buttons labels (#11841)
+  * Use crispEdges rendering for octicon-internal-repo (#11801)
+  * Work around fomantic 2.8.5 bugs (#11767)
+  * Show update branch item in merge box when it's necessary (#11761)
+  * Add compare link to releases (#11752)
+  * Allow site admin to disable mirrors (#11740)
+  * Export monaco editor on window.codeEditors (#11739)
+  * Update fomantic-ui to 2.8.5 (#11736)
+  * Refactor repo_stats to use os.pipe (#11726)
+  * bug: fix wrong poster message for code comment on Pull view (#11721)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 31]
```diff
@@ -4,6 +4,313 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix markdown meta parsing (#12817)
+  * Add default storage configurations (#12813)
+  * Show PR settings on empty repos (#12808)
+  * Disable watch and star if not signed in (#12807)
+  * Whilst changing the character set to utf8mb4 we should set ROW_FORMAT=dynamic too (#12804)
+  * Set opengraph attributes on org pages (#12803)
+  * Return error when creating gitlabdownloader failed (#12790)
+  * Add migration for password algorithm change (#12784)
+  * Compare SSH_DOMAIN when parsing submodule URLs (#12753)
+  * Fix editor.commit_empty_file_text locale string (#12744)
```
Comment (6543) [inline CHANGELOG.md:100]:
```suggestion
  * Fix editor.commit_empty_file_text locale string (#12744)
  * Fix wrong poster message for code comment on Pull view (#11721)
```

File: CHANGELOG.md [hunk 32]
```diff
@@ -4,6 +4,313 @@ This changelog goes through all the changes that have been made in each release
...
+  * OpenGraph: use repo avatar if exist (#12586)
+  * Reaction picker display improvements (#12576)
+  * Fix emoji replacements, make emoji images consistent (#12567)
+  * Increase clickable area on files table links (#12553)
+  * Set z-index for sticky diff box lower (#12537)
+  * Report error if API merge is not allowed (#12528)
+  * LFS support to be stored on minio (#12518)
+  * Show 2FA info on Admin Pannel: Users List (#12515)
+  * Milestone Issue/Pull List: Add octicons type (#12499)
+  * UI: Update commit signature view (#12473)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 33]
```diff
@@ -4,6 +4,313 @@ This changelog goes through all the changes that have been made in each release
...
+  * Set context for running CreateArchive to that of the request (#12555)
+  * Prevent redirect back to /user/events (#12462)
+  * Re-attempt to delete temporary upload if the file is locked by another process (#12447)
+  * Mirror System Notice reports are too frequent (#12438)
+  * Do not show arrows on comment diffs on pull comment pages (#12434)
+  * Fix milestone links (#12405)
+  * Increase size of the language column in language_stat (#12396)
+  * Use transaction in V102 migration (#12395)
+  * Only use --exclude on name-rev with git >= 2.13 (#12347)
+  * Update chroma to v0.8.0 (#12337)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 34]
```diff
@@ -4,6 +4,313 @@ This changelog goes through all the changes that have been made in each release
...
+  * Disable password complexity check default (#12557)
+  * Change PIDFile default from /var/run/gitea.pid to /run/gitea.pid (#12500)
+  * Add extension Support to Attachments (allow all types for releases) (#12465)
+  * Remove IE11 Support (#11470)
+* FEATURES
+  * Adopt repositories (#12920)
+  * Set setting.AppURL as GITEA_ROOT_URL environment variable during pushes (#12752)
+  * Check passwords against HaveIBeenPwned (#12716)
+  * Gitea 2 Gitea migration (#12657)
+  * Add postgres schema to the search_path on database connection (#12634)
```
[the remark this answers is not in the document]
Comment (zeripath) [reply CHANGELOG.md]: similarly

File: CHANGELOG.md [hunk 35]
```diff
@@ -4,6 +4,295 @@ This changelog goes through all the changes that have been made in each release
...
+  * Only use async pre-empt hack if go < 1.15 (#12718)
+  * Inform user about meaning of an hourglass on reviews (#12713)
+  * Add a migrate service type switch page (#12697)
+  * Migrations: Gitlab Add Reactions Support for Issues & MergeRequests (#12695)
+  * Remove duplicate logic in initListSubmits (#12660)
+  * Set avatar image dimensions (#12654)
+  * Rename models.ProtectedBranchRepoID/PRID to models.EnvRepoID/PRID and ensure EnvPusherEmail is set (#12646)
+    * Set setting.AppURL as GITEA_ROOT_URL environment variable during pushes (#12752)
+    * Add postgres schema to the search_path on database connection (#12634)
+  * Git migration UX improvements (#12619)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Set avatar image dimensions (#12654)
  * Rename models.ProtectedBranchRepoID/PRID to models.EnvRepoID/PRID and ensure EnvPusherEmail is set (#12646)
  * Set setting.AppURL as GITEA_ROOT_URL environment variable during pushes (#12752)
  * Add postgres schema to the search_path on database connection (#12634)
  * Git migration UX improvements (#12619)
```

File: CHANGELOG.md [hunk 36]
```diff
@@ -4,6 +4,295 @@ This changelog goes through all the changes that have been made in each release
...
+  * Don't add same line code comment box twice (#11837)
+  * Fix visibility of forked public repos from private orgs (#11717)
+  * Fix chardet test and add ordering option (#11621)
+  * Fix number of files, total additions, and deletions on Diff pages (#11614)
+  * Properly handle and return empty string for dangling commits in GetBranchName (#11587)
+  * Include query in sign in redirect (#11579)
+  * Fix Enter not working in SimpleMDE (#11564)
+  * Fix bug about can't skip commits base on base branch (#11555)
+* ENHANCEMENTS
+  * Save TimeStamps for Star, Label, Follow, Watch and Collaboration to Database (#13124)
```
Comment (6543) [inline CHANGELOG.md:144]:
```suggestion
  * Add HostCertificate to sshd_config in Docker image (#13143)
  * Save TimeStamps for Star, Label, Follow, Watch and Collaboration to Database (#13124)
```

**Review by zeripath — APPROVED** [review]

File: CHANGELOG.md [hunk 37]
```diff
@@ -4,6 +4,296 @@ This changelog goes through all the changes that have been made in each release
...
 
+## [1.13.0-RC1](https://github.com/go-gitea/gitea/releases/tag/v1.13.0-RC1) - 2020-10-14
+
+* SECURITY
+  * Use argon as default password hash algorithm (#12688)
+* BREAKING
+  * Mitigate Security vulnerability in the git hook feature (#13058)
+  * Disable DSA ssh keys by default (#13056)
+  * Don't replace underscores in auto-generated IDs in goldmark (#12805)
+  * Set TLS minimum version to 1.2 (#12689)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
* SECURITY
  * Mitigate Security vulnerability in the git hook feature (#13058)
  * Disable DSA ssh keys by default (#13056)
  * Set TLS minimum version to 1.2 (#12689)
  * Use argon as default password hash algorithm (#12688)
* BREAKING
  * Don't replace underscores in auto-generated IDs in goldmark (#12805)
```

**Comment by 6543** [comment]
@zeripath moved 12689,13056,13058 into sec topic - still ok?

**Comment by zeripath** [comment]
yeah fine

**Review by 6543 — COMMENTED** [review]
LGTM

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**